app/pages/patient/patientinfo.js

```diff
@@ -298,9 +297,10 @@ var PatientInfo = React.createClass({
     } else {
       return 'Birthdate not known';
```

Comment:
I know you didn't introduce this @GordyD but I never knew it was there, and it makes me cringe. (It's somewhat rare but possible for infants to be diagnosed with type 1 diabetes, so this is a possible use case for a parent creating an account to track their child's data.) Let's change to 'Born this year'? Or maybe add an `else if` for `yrsAgo === 0` returning 'Born this year' and keeping the `else` for the truly unknown edge cases?